README.md

```diff
@@ -62,3 +62,9 @@ Check out our [Official Documentation](https://kaos.ki-labs.com)
     <img src="docs/.gitbook/assets/image%20%288%29.png"
         height="75">
 </p>
+
+### Contributors
```

Comment:
please move Contributors *above* the logo so it fits the other sections (including the horizontal line - ##).

```
##
### Get Involved 
blah blah

##
### Contributors
<images>

LOGO
```